htlcswitch/link.go

```diff
@@ -1464,6 +1481,14 @@ func (l *channelLink) htlcManager() {
 						" %v", err),
 				)
 			}
+		case qReq := <-l.newQuiescenceRequests:
+			l.activeQuiescenceRequest = qReq
```

Comment:
Curious about the use case for allowing multiple incoming requests for quiescence, but only notifying the most recent request? Would have thought that we'd want to error out if we get a second request for quiescence (enforce one at a time) or return the result to all requestors?

Reply:
Maybe this code can be made more clear. I do think you're right that there's something fishy here. We don't want to allow concurrent requests.

Reply:
Yeah I think we should either return a global (relative to the channel link instance) channel for the `stfu`, or return an error if an stfu is already on going. 

Depends on what the future interaction is like between sub-systems that want to `stfu` the link.

Reply:
Also it's problematic that here we'll override the response channel, but then `initStfu` will return an error if we we're already attempting to cary out an stfu flow.

Reply:
Yeah I have fixed this at one level at this point. Gonna do a second pass to implement in terms of Result instead of Option.